specification/batch/data-plane/Microsoft.Batch/stable/2020-03-01.11.0/BatchService.json

```diff
@@ -15822,7 +15822,8 @@
         "destination",
         "uploadOptions"
       ],
-      "title": "A specification for uploading files from an Azure Batch Compute Node to another location after the Batch service has finished executing the Task process."
+      "title": "A specification for uploading files from an Azure Batch Compute Node to another location after the Batch service has finished executing the Task process.",
+      "description": "On every file upload, Batch service writes two log files to the compute node, fileuploadout.txt and fileuploaderr.txt.These log files is used to learn more about a specific failure."
```

Comment:
On every file upload Batch service writes two log files to the compute node, fileuploadout.txt and fileuploaderr.txt. These log files can be used to learn more about a specific failure.